### DISCUSSION TITLE
What is the best practice for `font name` definition on `skinparam` or `style`?

### DISCUSSION THREAD

**Discussion by The-Lum** [plantuml/plantuml#1264] [Q&A] [answered]
Hello PlantUML team,

- What is the best practice for `font name` definition on `skinparam` or `style`?
With or without quote or double-quote? according or not CSS-style?

For example:
```puml
skinparam defaultFontName       "Verdana"
```
Generate pseudo-malformed SVG, with:
```svg
...
<text fill="#FFFFFF" font-family="&quot;Verdana&quot;" font-size="12.5" ...
...
```

Thanks for your answer.
	
Ref.:
- new `styler parser`
- #1250
- https://github.com/plantuml/plantuml/commit/cc840383e3a8be213350b3e7142ac2491d6c7be6#diff-6d50778129cdb9f327928ba10313b69d72162fdf6f9e0707ff0e0981d7c8e749

**Comment by arnaudroques** [comment, marked as answer]
I think that the best option is to use double-quote.
We've [just fixed it here](https://github.com/plantuml/plantuml/commit/54afac2f270df882664f020b1971cf0db37218de).
Thanks for the report!
